lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -372,26 +382,21 @@ public function __toString()
 
     private function executeRegisteredSql($dryRun = false, $timeAllQueries = false)
     {
+        $executedQueries = 0;
```

Comment:
I thought of `executedRegisterSql` to return the number of actual SQL statements executed (considering dry run). That's the reason for this variable. However this can be implemented either by incrementing the variable in the loop (as is now), or getting the count after the loop using `count($this->sql)`. I don't have a strong opinion on which one would be better.